app/safe/internal/server/engine/server.go

```diff
@@ -8,7 +8,7 @@
 >/'  SPDX-License-Identifier: BSD-2-Clause
 */
 
-package server
+package engine
```

Comment:
@sahinakyol, I moved some of your code. Didn’t change the logic much.

It should still work, but I haven’t tested them.